kolibri/deployment/default/urls.py

```diff
@@ -24,7 +24,7 @@
 
 urlpatterns = [
     url(r'^admin/', include(admin.site.urls)),
-    url(r'^$', include('kolibri.core.urls')),
+    url(r'^kolibri/', include('kolibri.core.urls')),
```

Comment:
I don't think it's a problem to have a default deployment urlconf that puts `kolibri.core.urls` into `r'^/'`, i.e. the root instead of `r'^kolibri/'`. But we have to ensure that *someone* can move the whole thing somewhere else! That *someone* could even be ourselves doing a refactor! :)